source/index.html.md

```diff
@@ -679,6 +678,57 @@ object][ApplePay-PaymentToken] for more information.
   subsequent recurring authorization.
 </p>
 
+
+##### Method: `googlepay`
+
+To accept a payment using Google Pay, the complete payment token, recipient ID
+and derived shared secret, are required. Please refer to the [official
+documentation][GooglePay-PaymentCryptography]. Only protocol version `ECv2` is
+supported.
+
+<dl class="dl-vertical">
+  <dt>googlepay[token]
+    <span class="type">[:json:]</span>
+  </dt>
+  <dd>
+    Raw payment method token as received in response from Google. UTF-8 encoded
+    serialization of a JSON dictionary.
+  </dd>
+  <dt>googlepay[shared_key]
+    <span class="type">[:base64:]</span>
+  </dt>
+  <dd>
+    The shared secret derived from the ephemeral public key and your private
+    key.
+  </dd>
+  <dt>googlepay[recipient_id]
+    <span class="type">[\x20-\x7E]+
+      <a target="_blank" href="http://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters">ASCII printable characters excluding space</a>
+    </span>
+  </dt>
+  <dd>
+    The Google Pay recipient of the payload, e.g. <code>merchant:0123456789</code>.
+  </dd>
+</dl>
+
+<p class="alert alert-info">
+  <b>Notice:</b> Signing is required to use the <code>googlepay</code> payment
+  method.
+  <br />
+  <b>Notice:</b> An authorization made with <code>googlepay</code> may be fully
+  3-D Secured, 3-D Secure attempted, or with no 3-D Secure; this is indicated by
+  the <code>eciIndicator</code> of the <code>paymentMethodDetails</code>.
+  <br />
+  <b>Notice:</b> An authorization made with <code>googlepay</code> is
+  strongly authenticated (SCA in PSD2) if fully 3-D Secure.
```

Comment:
I think we need to relax this statement. As a random example, Mastercard LI `kA` has status `Y` and is not SCA.

Reply:
I think you're better equipped to answer that than me :)